chore(docs): title casing and grammar fixes (#18445)
gatsbyjs/gatsby: docs/docs/static-query.md

)
```

By using `StaticQuery`, you can colocate a component with its data. It is no longer required to, say, pass data down from `Layout` to `Header`.

### useStaticQuery

> There's also a hooks version of StaticQuery called `useStaticQuery`. [Check out its documentation here!](/docs/use-static-query/)

Reviewer: Here's a suggestion!
```suggestion
There's also a React hooks version of StaticQuery: check out the documentation on [`useStaticQuery`](/docs/use-static-query/)
```
